packages/std/src/init_handle.rs

```diff
@@ -11,22 +11,39 @@ use crate::types::HumanAddr;
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum CosmosMsg {
-    // this moves tokens in the underlying sdk
-    Send {
-        from_address: HumanAddr,
-        to_address: HumanAddr,
-        amount: Vec<Coin>,
-    },
+    Bank(BankMsg),
+    // this is dangerous to use, as it ties you to one particular runtime format.
+    // this makes the contract non-portable, and also fragile to break upon a hardfork
+    // only safe way is to receive it from a user and hold it temporarily.
+    Native { msg: Binary },
```

Comment:
Do we want to change to `Native(Binary)` or do we expect more than one proterty? I don't have any preference here, just noted it is a different type than the other enum cases.

Reply:
I kept it close to `Opaque { data: Binary }` but with a better name.

I don't know if there are any other sub-enums we can expect here. I will think on this over dinner, then make an adjustment.

Reply:
@webmaster128 what do you think about those two `NativeMsg` types? Or should I do that as a separate PR?

Reply:
I don't really understand the consequences on the Cosmos SDK side, but it would streamline the `Native` enum case with the other two

Reply:
I just used NativeMsg::Raw for now.
I have an idea to extend this in another PR.